src/somef/utils/constants.py

```diff
@@ -83,6 +119,7 @@
 CAT_USAGE = "usage"
 CAT_WORKFLOWS = "workflows"
 CAT_TYPE = "type"
+CAT_CONTINUOS_INTEGRATION= "continuousintegration"
```

Comment:
should be continuousIntegration for consistency.
Also, this is not added in the documentation. Please add it.